docs/cookbook/silver.module.md

```diff
@@ -1,20 +1,25 @@
 # silver.module
 
-There  is a mechanism to load specific controllers by using eZ content object. The eZ content type is "silver.module" to define a controller path and optionally parameters for this controller. This provides a convenient opportunity to define own, simple and translatable URLs using a silver.module name to any controller.
+silver.module is a special Content Type that enables you to load specific controllers using a Content item.
+silver.module defines a controller path and optionally parameters for this controller.
+This provides a convenient opportunity to define custom, simple and translatable URLs using a silver.module name to any controller.
 
 Following attributes are possible:
 
 |Attribute name|Description|Example value|
 |--- |--- |--- |
-|Name|The name of the silver.module. It is shown on the front end and is translatable.|My test module (results in an URL like: /my-test-module)|
-|Controller|The controller, which should be loaded when clicking on that module. Please define the controller including the fully qualified namespace and the controller method separated by two colons (::).</br>The silver.module controller handler, whichs loads the target controller automatically validates, if the controller class and action method is available and throws an exception otherwise.|\Silversolutions\Bundle\EshopBundle\Controller\CommonController::testAction|
-|Parameters|Optional parameters (list of hashes) which is directed to the target controller. Keep in mind, that only string key-value pairs are possible. The key value pairs are separated by a ";"|Parameter key: `formTypeResolver`</br>Parameter value: `contact`|
+|Name|The name of the silver.module. It is shown on the front end and is translatable.|"My test module", results in the URL: `/my-test-module`|
+|Controller|The controller which should be loaded when clicking that module. Define the controller including the fully-qualified namespace and the controller method separated by two colons (`::`). The silver.module controller handler which loads the target controller automatically validates if the controller class and action method are available and throws an exception otherwise.|`\Silversolutions\Bundle\EshopBundle\Controller\CommonController::testAction`|
```

Comment:
```suggestion
|Controller|The controller which should be loaded when clicking that module. Define the controller including the fully-qualified namespace and the controller method separated by two colons (`::`). The silver.module controller handler which loads the target controller automatically validates if the controller class and action method are available or throws an exception otherwise.|`\Silversolutions\Bundle\EshopBundle\Controller\CommonController::testAction`|
```